predictoor/pdr-faq.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Predictoor FAQ
+description: Frequently Asked Questions about Predictoor
+---
+
+## Predictoor FAQ
+<details>
+
+<summary>How Predictoor compare to prediction markets? Like Predictoor, they use \$ for accountability.</summary>
+
+Prediction markets are for one-off predictions like “will Biden win in 2024?” whereas Predictoor-style prediction feeds are for a continuous stream of predictions.
+
+</details>
+
+<details>
+
+<summary>How Predictoor compare to Chainlink? Like Predictoor, it has price feeds on-chain.</summary>
```

Comment:
`How Predictoor` -> `How does Predictoor`

Why: fix grammar. (BTW the blog had this error too. I just fixed it there too.)

Reply:
Cheers, done!